crates/bootstrap/Cargo.toml

```diff
@@ -19,7 +19,6 @@ image = { version = "0.24.2", default-features = false, features = ["png"] }
 log = "0.4.17"
 pollster = "0.3.0"
 profiling = "1.0.6"
-tracy-client = { version = "0.15.1", optional = true }
-
+tracy-client = { version = "0.17.0", optional = true }
+wgpu = "22.0.0"
```

Comment:
Why also upgrade tracy-client here? Also, can we keep the alphabetical sorting of dependencies?

Reply:
Will fix the ordering, and we accidentally included the updated tracy-client from our fork it seems, oops!